facturacion_mexico/cfdi_recibidos/services/purchase_invoice_builder.py

```
@@ -195,6 +195,89 @@ def _build_pi_doc(cfdi_doc):
+	"""
+	Resuelve expense_account según modo_resolucion_cuenta_gasto de la config.
+	Retorna el name de la Account, o None con advertencia en frappe.msgprint si no se encuentra.
+	Nunca asigna silenciosamente — si no hay cuenta, retorna None y registra advertencia.
+	"""
+	sat_code = f"{familia}.{sufijo.zfill(2)}"
+	modo = config.get("modo_resolucion_cuenta_gasto") or "manual_asistido"
+
+	if modo == "manual_asistido":
+		return None
+
+	if modo == "patron" or modo == "matriz_equivalencias":
```

Ruff (0.15.15)

[warning] Consider merging multiple comparisons: `modo in {"patron", "matriz_equivalencias"}`.

Merge multiple comparisons

(PLR1714)